kedro_airflow_k8s/cli.py

```diff
@@ -281,3 +208,22 @@ def name(tags: List[Dict[str, str]]) -> str:
     pipelines = [[name(d.tags), d.dag_id] for d in dags]
     pipelines.sort()
     click.echo(tabulate(pipelines, headers=["Name", "ID"]))
+
+
+@airflow_group.command()
+@click.option(
+    "-d",
+    "--dag-name",
+    "dag_name",
+    type=str,
+    required=False,
+    help="View for this specific DAG will be opened",
+)
+@click.pass_context
+def ui(ctx, dag_name: Optional[str] = None):
+    """Open Apache Airflow UI in new browser tab"""
+    context_helper = ctx.obj["context_helper"]
+    host = context_helper.airflow_config["airflow_ui_uri"]
```

Comment:
I think those two config params could be replaced with one `airflow_uri`. I don't expect API suffix`/api/v1` to change frequently so it requires another config entry

Reply:
Good point, I have now left only one. Additionally AirflowClient will add the REST API suffix, since the code logic is tied to specific REST API version.